modules/serverless-ossm-disabling-network-policies.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-ossm-setup.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-ossm-disabling-network-policies.adoc_{context}"]
+= Disabling the default network policies 
+
+To disable the default network policy generation, you can annotate the `KnativeEventing` and `KnativeServing` custom resources (CRs) with the `serverless.openshift.io/disable-istio-net-policies-generation` annotation.
+
+This prevents the operator from applying the default network policies.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving and Knative Eventing on your cluster.
+* You have installed {SMProductName} with the mTLS functionality enabled.
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
```

Comment:
This prereq can be broken down:

* You have one of the following permissions to access the cluster :
      ** cluster administrator permissions on {ocp-product-title} 
      ** cluster administrator permissions on {rosa-product-title}
      ** dedicated administrator permissions {dedicated-product-title}.

Reply:
Done!